admin/src/components/Wysiwyg/WysiwygNav.tsx

```diff
@@ -0,0 +1,266 @@
+              id="Italic"
+              label="Italic"
+              name="Italic"
+              icon={<Italic />}
+            />
+            <CustomIconButton
+              onClick={() => onActionClick('Underline', editorRef)}
+              id="Underline"
+              label="Underline"
+              name="Underline"
+              icon={<Underline />}
+            />
+          </MainButtons>
+
+          <MoreButton
+            ref={buttonMoreRef}
+            onClick={handleTogglePopover}
+            id="more"
+            label="More"
+            icon={<More />}
+          />
+          {visiblePopover && (
+            <Popover centered source={buttonMoreRef} spacing={4} id="popover">
+              <FocusTrap onEscape={handleTogglePopover} restoreFocus={false}>
+                <Flex>
+                  <IconButtonGroupMargin>
+                    <CustomIconButton
+                      onClick={() => onActionClick('Strikethrough', editorRef, handleTogglePopover)}
+                      id="Strikethrough"
+                      label="Strikethrough"
+                      name="Strikethrough"
+                      icon={<StrikeThrough />}
+                    />
+                    <CustomIconButton
+                      onClick={() => onActionClick('BulletList', editorRef, handleTogglePopover)}
+                      id="BulletList"
+                      label="BulletList"
+                      name="BulletList"
+                      icon={<BulletList />}
+                    />
+                    <CustomIconButton
+                      onClick={() => onActionClick('NumberList', editorRef, handleTogglePopover)}
+                      id="NumberList"
+                      label="NumberList"
+                      name="NumberList"
+                      icon={<NumberList />}
+                    />
+                  </IconButtonGroupMargin>
+                  <IconButtonGroup>
+                    <CustomIconButton
+                      onClick={() => onActionClick('Code', editorRef, handleTogglePopover)}
+                      id="Code"
+                      label="Code"
+                      name="Code"
+                      icon={<Code />}
+                    />
+                    <CustomIconButton
+                      onClick={() => {
+                        handleTogglePopover();
+                        onToggleMediaLib();
+                      }}
+                      id="Image"
+                      label="Image"
+                      name="Image"
+                      icon={<Image />}
+                    />
+                    <CustomLinkIconButton
+                      onClick={() => onActionClick('Link', editorRef, handleTogglePopover)}
+                      id="Link"
+                      label="Link"
+                      name="Link"
+                      // eslint-disable-next-line jsx-a11y/anchor-is-valid
+                      icon={<Link />}
```

Comment:
Inlined event handlers and inlined icon elements will triggered re-renders. Handler should be wrapped with `useCallback` and icons moved outside of the component. For example

```tsx
const linkIcon = <Link />;

// ...

const WysiwygNav: React.FC<WysiwygNavProps> = ({
```

Consider moving all icons declarations to other file in the same directory `icons.tsx`.